packages/wordcount/src/test/index.test.js

```diff
@@ -43,10 +43,17 @@ describe( 'WordCounter', () => {
 		{
 			message: 'Punctuation.',
 			string: 'It’s two three \u2026 4?',
-			words: 3,
+			words: 4,
 			characters_excluding_spaces: 15,
 			characters_including_spaces: 19,
 		},
+		{
+			message: 'Numbers as word',
+			string: 'Should be 4 words',
```

Comment:
I did a quick cross-check with Pages from Apple and it produces the same result:

<img width="348" alt="Screen Shot 2021-05-14 at 09 29 45" src="https://user-images.githubusercontent.com/699132/118236949-feed7100-b496-11eb-87df-1e98ee584cdd.png">

Reply:
I see also a comment from @Soean in https://github.com/WordPress/gutenberg/issues/26339#issuecomment-717069789:

> Good point, I tested Microsoft Word, Google Docs and Libre office, all apps count numbers as a word. So I think we should change it in WordPress